LinqToRest.Test/PluralizerTest.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Xunit;
+
+namespace Messerli.LinqToRest.Test
+{
+    public class PluralizerTest
```

Comment:
Seal this class

Reply:
https://github.com/messerli-informatik-ag/linq-to-rest/pull/38/commits/3f1134df3175182b788de85153e1b4dbec0712ca